style.css

```diff
@@ -0,0 +1,675 @@
+  margin-top: 20px;
+}
+
+.featured-professors .professors-list .professor .img-container {
+  background: url(assets/peakpx.jpg);
+  background-repeat: no-repeat;
+  background-size: 80px 80px;
+  margin-right: 20px;
+}
+
+.partners .partner-icons-list .partner-icons-list-child li img {
+  margin-left: 30px;
+}
+
+.featured-professors .professors-list .professor .img-container img {
+  padding-left: 15px;
+  padding-top: 20px;
+  object-fit: cover;
+  width: 140px;
+  height: 140px;
+}
+
+.featured-professors .professors-list .professor .professor-name {
+  font-family: "Cocogoose", sans-serif;
+  font-size: 14px;
+  margin-bottom: 10px;
+}
+
+.featured-professors .professors-list .professor .professor-summary {
+  font-family: "Cocogoose-Italic", sans-serif;
+  font-size: 8px;
+  margin-bottom: 20px;
+  color: #ec5242;
+}
+
+.featured-professors .professors-list .professor hr {
+  width: 12px;
+  color: #757370;
+  margin-bottom: 20px;
+}
+
+.featured-professors .professors-list .professor .professor-studies {
+  font-family: "Lato", sans-serif;
+  font-size: 8px;
+  color: #4a4848;
+}
+
+.featured-professors .more-professors-btn {
+  margin-bottom: 30px;
+  margin-top: 30px;
+  padding: 20px 30px;
+  background-color: #fff;
+  width: 280px;
+  align-self: center;
+  font-family: "Lato", sans-serif;
+  font-size: 18px;
+}
+
+header .about-header-container {
+  display: flex;
+  flex-direction: column;
+  margin-top: 35px;
+  text-align: center;
+  padding-bottom: 50px;
+}
+
+.about-header-container .about-intro {
+  font-family: "cocogoose-trial", sans-serif;
+  text-transform: uppercase;
+  display: block;
+  margin-top: 30px;
+  color: rgba(236, 82, 66, 1);
+  -webkit-background-clip: text;
+  font-size: 40px;
+  text-align: center;
+  align-self: center;
+  max-width: 500px;
+}
+
+.about-header-container .about-intro-paragraph {
+  border: 2px solid white;
+  background-color: #d3d3d3;
+  padding: 15px;
+  font-family: "Lato", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  color: #4a4848;
+  margin-top: 30px;
+  text-align: center;
+  align-self: center;
+  max-width: 500px;
+}
+
+.about-header-container .about-contact-p {
+  font-family: "Cocogoose", sans-serif;
+  font-size: 10px;
+  color: #4a4848;
+  margin-top: 25px;
+  text-align: center;
+}
+
+.about-header-container .about-email {
+  font-family: "Lato", sans-serif;
+  font-size: 18px;
+  color: #4a4848;
+  margin-top: 25px;
+  text-align: center;
+  text-decoration: underline;
+}
+
+.logo-section {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 25px;
+}
+
+.logo-section .logo-section-head {
+  font-family: "Lato", sans-serif;
+  font-size: 28px;
+  color: #292828;
+  margin-top: 25px;
+  text-align: center;
+  max-width: 500px;
+}
+
+.logo-section .logo-section-desc {
+  font-family: "Lato", sans-serif;
+  font-size: 20px;
+  color: #4a4848;
+  margin-top: 25px;
+  text-align: center;
+  max-width: 500px;
+}
```

Comment:
- [ ] Kindly make sure to adjust the typography of this paragraph in order to maat the provided Behance design as much as possible.

   - ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/225582349-b76f3a8b-b6bf-4af6-aac2-0bc8cc4c27f7.png)

   - ### BEHANCE DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/225582431-fabd3f58-dca6-4d23-aec9-9a9383866e99.png)